awx_collection/tests/integration/targets/inventory/tasks/main.yml

```
@@ -193,7 +193,9 @@
         that:
           - "result is failed"
           - "result is not changed"
-          - "'test-non-existing-org' in result.msg"
+          - >-
+            'Request to /api/v2/organizations/?name=test-non-existing-org' in result.msg and
```

```suggestion
            'test-non-existing-org' in result.msg and
```

I'd prefer to leave out the url endpoint, as urls can change or have a different api prefix

---

That makes sense, thank you so much!